demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -33,30 +34,90 @@ def build_argparser():
                       help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
                            "acceptable. The demo will look for a suitable plugin for device specified. "
                            "Default value is CPU", default="CPU", type=str)
-    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8, type=int)
-    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask.", default=24, type=int)
-    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask.", default=100, type=int)
-    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask.", default=20, type=int)
-    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+    args.add_argument("-r", "--rnd", help="Optional. Use random mask for inpainting (with parameters set by -p, -mbw, -mk and -mv).Skipped in GUI mode", action='store_true')
+    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask. Skipped in GUI mode", default=8, type=int)
+    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask. Skipped in GUI mode", default=24, type=int)
+    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask. Skipped in GUI mode", default=100, type=int)
+    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask. Skipped in GUI mode", default=20, type=int)
+    args.add_argument("-mc", "--mask_color", help="Optional. Color to be treated as mask (provide 3 RGB components in range of 0...255). Default is 0 0 0. Skipped in GUI mode", default=0, type=int, nargs="+")
+    args.add_argument("--no_show", help="Optional. Don't show output. Skipped in GUI mode", action='store_true')
+    args.add_argument("-o","--output", help="Optional. Save output to the file with provided filename. Skipped in GUI mode", default="", type=str)
+    args.add_argument("-a", "--auto", help="Optional. Use automatic (non-interactive) mode instead of GUI", action='store_true')
 
     return parser
 
 def main():
     args = build_argparser().parse_args()
 
-    ie = IECore()
-    inpainting_processor = ImageInpainting(ie, args.model, args.parts,
-                                           args.max_brush_width, args.max_length, args.max_vertex, args.device)
+    if args.auto:
+        # Command-line inpaining for just one image
+        concat_image,result = inpaintRandomHoles(args)
+        if args.output!="":
+            cv2.imwrite(args.output,result)
+        if not args.no_show:
+            cv2.imshow('Image Inpainting Demo', concat_image)
+            cv2.waitKey(0)
+    else:
+        # GUI inpainting
+        gui = InpaintingGUI(args.input,args.model, args.device)
+        gui.run();
+
+def inpaintRandomHoles(args):
```

Comment:
Move functions (this and `createRandomMask`) upper for correct logic "definition" - "usage".

Reply:
Done